scripts/tasks/docs-reorg.json

```diff
@@ -0,0 +1,80 @@
+{
+  "docs/experimental.md": "docs/api/experimental.md",
```

Comment:
This is a non-blocking comment, but I'm wondering if re-organizing the contents of the `/docs` folder is truly necessary for our purposes. Doing this will break existing URLs, and is not necessary for the sidebar configuration.

I guess it would make navigating through the docs on GitHub a bit easier?

Reply:
>  is truly necessary for our purposes.

My preference would be to have the content reorganization upstream so developers have an easier way finding things and knowing where things should go. If we don't use the paths, then we will have to force contributors to use a particular property in frontmatter indicating the category.

> Doing this will break existing URLs,

Maybe we can look at the current analytics and see how much direct traffic goes to those pages. If it's significant we can probably create a redirect . TBH, I'd rather have something meaningful in the URL than putting everything under `tutorial`:

![screenshot of the code signing document path locally: https://localhost:3000/docs/distribution/code-signing](https://user-images.githubusercontent.com/606594/115617199-f940a900-a2a5-11eb-9448-2401a309549d.png)

![screenshot of the code signing document in production: https://www.electronjs.org/docs/tutorials/code-signing](https://user-images.githubusercontent.com/606594/115617358-29884780-a2a6-11eb-8e36-14d5e23ae441.png)

> I guess it would make navigating through the docs on GitHub a bit easier?

Once we upstream the new location it should make it easier to navigate and to contribute.